### ISSUE TITLE
Smart selection of quote token

### ISSUE THREAD

**Issue by Velenir** [gnosis/dex-react#1273]
Smart selection of quote token and base token

Rafa's suggestion for simple and good enough solution is:
- We need to know when a token is a stable coin
- If it's stable coin vs non stable coin, we choose stable coin as a quote
- In any other case, we continue as now: sell is the quote

With the above, we can eliminate the need of showing the 2 prices.
We can, just in case allow to invert the price anyways.

_Estimation: 3.5_

**Comment by anxolin** [comment]
https://docs.google.com/document/d/1du5mgEQ1z34d2uJcXTii40U2MXWDcB9kViGDqqThFeI/edit?ts=5f242902#  @Rafanator

**Comment by anxolin** [comment]
Once we have a prototype for this, we can revalidate with Cryptocat, Rafa, and other traders to make sure the change is good

**Comment by Rafanator** [comment]
Feedback from Crypcat, notable DEX trader, stated that he prefers the selection previously discussed. 

```
CryptoCat, [3. Aug 2020 at 11:39:21]:
show weth always quoted in stablcoins
Having 0.0033565... prices when dealing with weth/stablcoin markets is annoying.

stable/stable let the user decide
```

This makes me once more prefer doing a preselection in the categories that I mentioned before:

1. Volatile/stable -> Always stable as quote
2. stable/stable -> As user inputs, respecting base/quote system
3. volatile/WETH -> WETH as quote token
4. volatile/volatile -> As user inputs 

@anxolin Thoughts? We should validate this with more traders.

**Comment by W3stside** [comment]
Closed by #1476
